recipes/pygit2/meta.yaml

```diff
@@ -0,0 +1,69 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "pygit2" %}
+{% set version = "0.25.0" %}
+{% set sha256 = "de0ed85fd840dfeb32bcaa94c643307551dc0d967c3714e49087e7edc0cdc571" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc9    # [win and py27]
+    - vc10   # [win and py34]
+    - vc14   # [win and py35]
+
+requirements:
+  build:
```

Comment:
Also, add this to the build requirements (I guess it is to be prepared for some future changes):

```
    - vc 9    # [win and py27]
    - vc 10   # [win and py34]
    - vc 14   # [win and py35]
```